components/migrate/steps/SignLoader1.vue

```diff
@@ -114,6 +115,11 @@ const fromCollection = collections.value.find(
 )
 
 const startStep1 = async () => {
+  if (disabledOperationPrefixes(from)) {
```

Comment:
What is step 1

Reply:
step1
<img width="344" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/5ea4a387-9b8e-41d6-9062-1c096983c8af">

I'm thinking about whether we should disable the `/migrate` page as they are only for `rmrk/ksm`

Reply:
> I'm thinking about whether we should disable the /migrate page

yup

Reply:
@Jarsen136 is this implemented ?